### ISSUE TITLE
git.mozilla.org https clone urls don't validate

### ISSUE THREAD

**Issue by justdave** [mozilla/contribute.json#58]
Reference:
https://www.bugzilla.org/contribute.json

The repo clone url listed there comes up as invalid when I try to run this on the validator on https://contribute.paas.allizom.org , but I assure you it is correct:

dave@rmbp:temp $ git clone https://git.mozilla.org/bugzilla/bugzilla.git
Cloning into 'bugzilla'...
remote: Counting objects: 93013, done.
remote: Compressing objects: 100% (23526/23526), done.
remote: Total 93013 (delta 67833), reused 89185 (delta 65193)
Receiving objects: 100% (93013/93013), 19.04 MiB | 5.64 MiB/s, done.
Resolving deltas: 100% (67833/67833), done.
Checking connectivity... done.

**Comment by fwenzel** [comment]
That URL is a 404 from here. Is there a routing problem and you're on a VPN or something?

**Comment by justdave** [comment]
Nope, but it only works in a git client, it doesn't work in a web browser.  I don't know why, that's just the way it works.  (I know little about the internals of git.m.o)

**Comment by peterbe** [comment]
There's nothing wrong with the URL. The validator sees the `https://` and thinks it should be a regular browser URL. 
I guess we need to make it exempt on urls that look like GitHub HTTPS urls.

**Comment by fwenzel** [comment]
> like GitHub HTTPS urls

Except it isn't ;)
